### PR TITLE
Feature/group environments

### PR DESCRIPTION
**Pull request by teiling88**

### CODE REVIEW

File: src/ScriptRuntime/CommandBuilder.php [hunk 1]
```diff
@@ -45,8 +45,7 @@ private function reset()
                 $this->currentShellCommand,
                 $this->startLine,
                 $this->ignoreError,
-                $this->tty,
-                $this->template
```
Comment (JanPietrzyk) [inline src/ScriptRuntime/CommandBuilder.php:49]: verry good catch!

File: src/ScriptRuntime/ScriptLoader.php [hunk 2]
```diff
@@ -119,11 +119,13 @@ private function findInclude(Script $fromScript, string $includeStatement): stri
      */
     private function isExecutableLine(string $command): bool
     {
-        if (!strlen(trim($command))) {
+        $command = trim($command);
```
Comment (JanPietrzyk) [inline src/ScriptRuntime/ScriptLoader.php:122]: like!

File: src/ScriptRuntime/TemplateEngine.php [hunk 3]
```diff
@@ -46,8 +46,8 @@ private function getValue(string $placeholder, array $allValues)
     {
         $valueName = substr($placeholder, 2, -2);
 
-        foreach ($allValues as $key => $value) {
-            if (strtoupper($key) == $valueName) {
+        foreach (array_keys($allValues) as $key) {
```
Comment (JanPietrzyk) [inline src/ScriptRuntime/TemplateEngine.php:49]: win!

**Review by JanPietrzyk — APPROVED** [review]

**Comment by JanPietrzyk** [comment]
@teiling88 could you please extend the `Shopware\Psh\Test\Acceptance\ApplicationTest` file to validate solidify the functionality introduced here?

**Comment by teiling88** [comment]
Done! :-)

**Review by JanPietrzyk — APPROVED** [review]

**Pull request merged**